projects/gnutls/build.sh

```diff
@@ -15,8 +15,15 @@
 #
 ################################################################################
 
+CONFIGURE_FLAGS=""
+if [[ $SANITIZER_FLAGS = "*sanitize=memory*" ]]
```

Comment:
👍  (FYI, I cribbed this from `boringssl`, so you may want to update that as well)